### PR TITLE
fix headers in Fireworks/Core

### PR DESCRIPTION
**Pull request by mrodozov**
Fix headers related to #15248

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: Fireworks/Core/src/FWGUIEventDataAdder.h [hunk 1]
```diff
@@ -22,6 +22,7 @@
 #include <set>
 #include <vector>
 #include <string>
+#include <TGFrame.h>
```
Comment (smuzaffar) [inline Fireworks/Core/src/FWGUIEventDataAdder.h]: @mrodozov , isn't forward declaration good enough in this case?

Comment (mrodozov) [reply Fireworks/Core/src/FWGUIEventDataAdder.h]: It is pr is modified.

**Comment by mrodozov** [comment]
please test

**Comment by alja** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**